testplan/exporters/testing/http/__init__.py

```diff
@@ -92,16 +93,15 @@ def _upload_report(
 
         return response, errmsg
 
-    def export(self, source) -> Union[None, str]:
-        """TODO"""
+    def export(self, source: TestReport) -> Union[None, str]:
```

Comment:
Do we have preference atm between Union[None, ...] and Optional[...]? Just dropping it in, reading up the PEPs there is no clear recommendation and both will move to, eventually, to the newer type annotation format with operators.

Reply:
TBH, I just copied around whatever was there already. But I like the Optional more, lets go with that.